shared/src/PayloadSerializer.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Azure.Devices.Shared
+{
+    /// <summary>
+    /// Provides the serialzation for a specified convention
```

Comment:
punctuation

Reply:
There are many other cases too. Remember, this will show up in web documentation, so each sentence should end with punctuation.

Reply:
I wish there was a plugin that we could use for this

Reply:
One of the many spell checkers may do it.